docs/tutorial.md

```diff
@@ -799,14 +799,13 @@ define a private flag:
 1. Define the flag value. This is normally defined as a macro that
   expands to a binary flag, for example: `#define MY_PRIVATE_FLAG (1 <<
   2)`.  Note that this will be part of the ABI, so it must be versioned
-1. Call `fu_device_register_private_flag` in the device init function
+1. Call `fu_device_register_custom_flag` in the device init function
```

Comment:
@superm1 i keep flip flopping between `fu_device_register_private_flag`, `fu_device_register_custom_flag` and `fu_device_register_internal_flag`. Maybe *internal* flag is better than _custom_ flag?

Reply:
I went with fu_device_register_private_flag... :)